snuba/datasets/outcomes_raw.py

```diff
@@ -63,3 +66,9 @@ def get_extensions(self) -> Mapping[str, QueryExtension]:
 
     def get_prewhere_keys(self) -> Sequence[str]:
         return ["project_id", "org_id"]
+
+    def get_query_processors(self) -> Sequence[QueryProcessor]:
+        return [
+            BasicFunctionsProcessor(),
+            PrewhereProcessor(),
```

Comment:
yes, forgetting this was dumb

Reply:
Not really, I think that having to repeat the prewhere processor in every dataset is a bit of a smell. It might not even belong here long term since it's really a storage layer optimization and not something that is really pertinent to the `Query` (as opposed to the `ClickhouseQuery`.)